src/components/ArticleV3/SimpleArticleCell.js

```diff
@@ -30,21 +30,36 @@ const ArticleCell = ({
   ready,
 }) => {
   const [isEditing, setIsEditing] = React.useState(false)
+  const [currentCell, setCurrentCell] = React.useState(useExecutionScope.getState().cells[idx])
 
-  const { executing, errors, outputs, thebeCell, executeCell, clearCell, resetCell } =
-    useExecutionScope((state) => ({
-      executing: state.cells[idx]?.executing ?? false,
-      errors: state.cells[idx]?.errors,
-      outputs: state.cells[idx]?.outputs ?? [],
-      thebeCell: state.cells[idx]?.thebe,
-      executeCell: () => state.executeCell(idx), // curried to this cell idx
-      clearCell: () => state.clearCell(idx),
-      resetCell: () => state.resetCell(idx),
-    }))
-
-  const updateCellSource = useExecutionScope((state) => state.updateCellSource)
-
-  console.log(type, outputs, thebeCell?.outputs)
+  const executing = !!currentCell?.executing
+  const errors = currentCell?.errors
+  const outputs = currentCell?.outputs || []
+  const thebeCell = currentCell?.thebe
+  // Connect to the store on mount, disconnect on unmount, catch state-changes in a reference
+  React.useEffect(
+    () =>
+      useExecutionScope.subscribe((state) => {
+        const cell = state.cells[idx]
+        if (!cell) return
+        if (cell.executing !== !!currentCell?.executing) {
```

Comment:
@danieleguido with this line in place intial outputs don't render and execution is not working.